synfig-studio/src/gui/dials/zoomdial.cpp

```diff
@@ -58,10 +58,10 @@ using namespace studio;
 ZoomDial::ZoomDial(Gtk::IconSize & size):
 	Gtk::Grid()
 {
-	zoom_in = create_icon(size, Gtk::Stock::ZOOM_IN, _("Zoom In"));
-	zoom_out = create_icon(size, Gtk::Stock::ZOOM_OUT, _("Zoom Out"));
-	zoom_fit = create_icon(size, Gtk::Stock::ZOOM_FIT, _("Zoom to Fit"));
-	zoom_norm = create_icon(size, Gtk::Stock::ZOOM_100, _("Zoom to 100%"));
+	zoom_in = create_icon(size, "gtk-zoom-in", _("Zoom In"));
```

Comment:
`gtk-zoom-in` -> `zoom-in`
`gtk-zoom-out` -> `zoom-out`
`gtk-zoom-100` -> `zoom-original` (https://docs.gtk.org/gtk3/const.STOCK_ZOOM_100.html)
`gtk-zoom-fit` -> `zoom-fit-best` (https://docs.gtk.org/gtk3/const.STOCK_ZOOM_FIT.html)

Reply:
done